.github/workflows/product-tests.yml

```diff
@@ -57,5 +57,6 @@ jobs:
           rm -rf ~/.m2/repository
       - name: Product Tests
         run: |
+          export PTL_BIND_PORTS=false
```

Comment:
let's this default behaviour. When needed for manual tests, user could configure launcher on their own.

Also let's not use env variable for that, but regular cli parameter.

Reply:
i want `env up` to expose ports by default and i didn't find a better way to achieve this